src/dataflow/src/render/join/delta_join.rs

```diff
@@ -181,25 +182,20 @@ impl DeltaJoinPlan {
     }
 }
 
-impl<G> Context<G, MirRelationExpr, Row, Timestamp>
+impl<G> Context<G, MirRelationExpr, Row, repr::Timestamp>
 where
-    G: Scope<Timestamp = Timestamp>,
+    G: Scope<Timestamp = repr::Timestamp>,
```

Comment:
just curious - was there a specific reason we decided to not import `Timestamp` at the top level and use `repr::Timestamp` everywhere?

Reply:
The other half of the PR (from a month back) is deleting references to `repr::Timestamp`. :)